greensnow-feed/operations.py

```diff
@@ -0,0 +1,66 @@
+"""
+Copyright start
+MIT License
+Copyright (c) 2024 Fortinet Inc
+Copyright end
+"""
+
+import requests
+from connectors.core.connector import get_logger, ConnectorError
+
+logger = get_logger('greensnow-feed')
+
+
+class GreenSnowFeed(object):
+    def __init__(self, config, *args, **kwargs):
+        self.url = config.get('server_url').strip('/')
+        self.verify_ssl = config.get('verify_ssl')
+
+    def make_rest_call(self, url, method, data=None, params=None):
```

Comment:
This method doesn't use the URL parameter can we remove it?
And the parameter method can have a default value 'GET'